.github/workflows/maven-cd.yml

```diff
@@ -0,0 +1,65 @@
+# Note: additional setup is required, see https://www.jenkins.io/redirect/continuous-delivery-of-plugins
```

Comment:
`.github/workflows/maven-cd.yml` not sure if `maven-cd` is the best filename for this workflow

Reply:
i think is fine

Reply:
although gradle could likely be supported in the same workflow, the actions would just need a flag, it just doesn't support it atm

Reply:
Gradle CD would need to use a different versioning scheme and it could follow a different flow. I would stick to `maven-cd`. If and when some Gradle user wishes to introduce their own reusable CD flow, they can figure out how to refactor things to share parts of this one, or not.